packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -130,13 +130,30 @@ function useHasTextDecorationControl( name, element ) {
 	return supports.includes( 'textDecoration' );
 }
 
+function TypographyToolsPanel( { ...props } ) {
+	return <ToolsPanel label={ __( 'Typography' ) } { ...props } />;
+}
+
+const DEFAULT_CONTROLS = {
+	fontFamily: true,
+	fontSize: true,
+	fontAppearance: true,
+	lineHeight: true,
+	letterSpacing: true,
+	textTransform: true,
+	textDecoration: true,
+};
+
 export default function TypographyPanel( {
+	as: Wrapper = TypographyToolsPanel,
 	name,
 	element,
 	value,
 	onChange,
-	inherit,
+	inherit = value,
```

Comment:
We've been wanting to access the inherited value for a long time now in the block inspector, this will actually make it super easy now to implement.